packages/application-components/src/theming.ts

```diff
@@ -0,0 +1,25 @@
+import kebabCase from 'lodash/kebabCase';
+import { designTokens } from '@commercetools-uikit/design-system';
+
+export const themesOverrides = {
+  default: {
+    paddingVerticalForPage: designTokens.spacingM,
+    paddingHorizontalForPage: designTokens.spacingM,
+  },
+  test: {
+    paddingVerticalForPage: designTokens.spacingXl,
+    paddingHorizontalForPage: '40px',
+  },
+};
+
+export const appKitDesignTokens = Object.values(themesOverrides).reduce(
```

Comment:
Here is where we create the design tokens to be used in the components styling code.
It _knows_ how the `ThemeProvider` will convert the tokens names into CSS variables so it does the same.

Reply:
We should probably expose the `toVar` function from the `design-system` package.